tapdb/universe_stats.go

```diff
@@ -53,6 +68,16 @@ type UniverseStatsStore interface {
 	// universe/
 	QueryUniverseAssetStats(ctx context.Context,
 		arg UniverseStatsQuery) ([]UniverseStatsResp, error)
+
+	// QueryAssetStatsPerDaySqlite returns the stats for a given asset
+	// grouped by day in a SQLite specific format.
+	QueryAssetStatsPerDaySqlite(ctx context.Context,
+		q AssetStatsPerDayQuery) ([]AssetStatsPerDay, error)
```

Comment:
Perhaps `AssetStatsPerDay` should be renamed to be sqlite specific to match the pattern for `AssetStatsPerDayPg`.

Reply:
The two query and response structs are identical, but `sqlc` creates unique ones anyway. And since in Golang we can't just convert one into the other easily (AFAIK), we need to use both. I chose one of them as the return type for the exported method (the API surface of the `tapdb` package), so it would be confusing if that had "Sqlite" in its name.
The alternative would be to have a third alias type, but that seemed overly excessive.